AmplifyPlugins/Core/AWSPluginsCore/Auth/AWSAuthService.swift

```diff
@@ -42,6 +42,40 @@ public class AWSAuthService: AWSAuthServiceBehavior {
         return validResult
     }
 
+    public func getTokenClaims() -> Result<[String: AnyObject], AuthError> {
```

Comment:
from your investigation, is there anything you can capture as a comment on top of this method or updated description in the PR to point to some meaningful resources related to this algorithm?

Reply:
added comment.